### DISCUSSION TITLE
Hosting verdaccio on netifly ??

### DISCUSSION THREAD

**Discussion by nivekithan** [verdaccio/verdaccio#2322] [Q&A] [answered]
Hi,

First of all I am frontend developer so apologies if my question makes no sense.

I am wondering is it possible to host verdaccio (only server not frontend) on netlify or heroku or any other free hosting service. 

I am participating in a hackathon where my plan is to create a code marketplace where developers and companies can buy and sell packages.

To host all these packages I am planning to use verdaccio for package registry with some custom plugins.

But I am not sure whats the process for hosting it.

Thanks for the help

**Comment by juanpicado** [comment, marked as answer]
hi @nivekithan 

Veradaccio is just an express app with some local file requirements, but is just a regular Node.js application.  Maybe google by "express app on Netifly" could give you a broad list of sources like, 

https://stackoverflow.com/questions/59868667/how-to-deploy-express-js-server-to-netlify

Personally I've never tried myself or heard someone sharing experiences.

**Reply by nivekithan** [reply]
Thanks @juanpicado  I will look into it
